charts/rollups-node/templates/NOTES.txt

```diff
@@ -10,7 +10,7 @@
   echo http://$SERVICE_IP:4000/graphql
 {{- else if contains "ClusterIP" .Values.validator.service.type }}
   export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "validator.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
-  export CONTAINER_PORT=$(kubectl get pod --namespace {{ .Release.Namespace }} $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
-  echo "Visit http://127.0.0.1:4000/graphql to use your application"
+  export CONTAINER_PORT={{ .Values.validator.service.port }}
```

Comment:
The CONTAINER_PORT should come from the `kubectl` output.

Also, there are still other references to the old 4000 port, if you could tackle them as well.

Reply:
> Also, there are still other references to the old 4000 port, if you could tackle them as well.

My change is related to the local port in this line of file : 

`  kubectl --namespace {{ .Release.Namespace }} port-forward $POD_NAME 8080:$CONTAINER_PORT`

The instructions will expose whatever the `graphql` port is to `8080` of localhost.